README.md

```diff
@@ -36,8 +36,8 @@ Contents:
 ### Prerequisites
 
 You must install:
-- A [supported version of Node](https://github.com/neon-bindings/neon#platform-support).
-- [Rust](https://www.rust-lang.org/tools/install) 
+- A [supported version of Node](https://github.com/neon-bindings/neon#platform-support);
```

Comment:
This should end in a period, per Adobe Style Guide https://writing.adobe.com/58b352410/p/646bcb-common-questions/t/186612

Reply:
Fixed!